tests/nat/eval/red_teaming_evaluator/fake_chat_models.py

```
@@ -0,0 +1,116 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import json
+from typing import Any
+from typing import Union # noqa: F401
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)